integration/getting_started_with_gossiped_ring_test.go

```diff
@@ -22,6 +22,14 @@ import (
 )
 
 func TestGettingStartedWithGossipedRing(t *testing.T) {
+	runTestGettingStartedWithGossipedRing(t, "series_1", generateFloatSeries)
```

Comment:
Can we make this like the others to not increase the tests time ? I like how in the other getting started tests we do the environment setup once, and then tests floats and histograms with the same setup.